src/warp10-datasource.ts

```diff
@@ -90,10 +90,17 @@ export default class Warp10Datasource {
         return { data }
       })
       .catch((err) => {
-        console.warn('[Warp 10] Failed to execute query', err)
+        let headers = err.headers();
+        let wsHeadersOffset: number = wsHeader.split('\n').length;
+        let errorline: number = Number.parseInt(headers['x-warp10-error-line']) - wsHeadersOffset;
```

Comment:
`try {} catch {}` because __parseInt__ can fail

Reply:
By construction in Warp 10, it cannot fail, as long as it is really here... 
There is a way to set a different header name in Warp 10 configuration, but it is not exposed.
To be sure, I will check the existence of both headers before.